### PR TITLE
doc: clarify prerequisites

### PR DESCRIPTION
**Pull request by monperrus**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -39,9 +39,22 @@ https://hacks.mozilla.org/2019/04/teaching-machines-to-triage-firefox-bugs/
 - **uplift** - The aim of this classifier is to detect bugs for which uplift should be approved and bugs for which uplift should not be approved.
 
 
-## Setup
+## Setup and Prerequisites
 
-Run `pip install -r requirements.txt` and `pip install -r test-requirements.txt`. Depending on the parts of bugbug you want to run, you might need to install dependencies from other requirement files (find them with `find . -name "*requirements*"`).
+libgit2 (needs [v1.0.0](https://github.com/libgit2/libgit2/releases/tag/v1.0.0), only in [experimental on Debian](https://wiki.debian.org/DebianExperimental))
```
Comment (marco-c) [inline README.md:44]: I'm using 0.28.5+dfsg.1-1 from Debian testing, did that not work for you?

File: README.md [hunk 2]
```diff
@@ -72,6 +85,11 @@ Note: If you run into problems, it's possible the version of Mercurial you are u
 
 Note: the script will take a long time to run (on my laptop more than 7 hours). If you want to test a simple change and you don't intend to actually mine the data, you can modify the repository.py script to limit the number of analyzed commits. Simply add `limit=1024` to the call to the `log` command.
 
+### Training the defect model
+
+To train the model for mode `defect`:
+
+    python3 -m scripts.trainer defect
```
Comment (marco-c) [inline README.md:92]: Could you move this to the `Usage` section? And maybe rename `Usage` to something like `Usage (training a model)`.

**Comment by marco-c** [comment]
Thanks! I've failed to keep the README updated, I'll try to make some additional changes in the next few days.

File: README.md (hunk 1 above)
Comment (monperrus) [reply README.md:44]: No, that failed at `pip3 install`, where one package does native compilation (probably pygit2), with an error message stating that v0.99 or v1.0.0 was required.

File: README.md (hunk 2 above)
Comment (monperrus) [reply README.md:94]: done

File: README.md (hunk 1 above)
Comment (marco-c) [reply README.md:44]: Strange, maybe for some reason pip is building it from source for you, and installing the wheel for me.

File: README.md [hunk 3]
```diff
@@ -54,7 +67,17 @@ Every time you will try to commit, pre-commit will run checks on your files to m
...
+### Usage (generic)
+
+Run the `trainer.py` script with the command `python3 -m scripts.trainer` (with `--help` to see the required and optional arguments of the command) to perform training (warning this takes 30min+).
+
+### Usage for mode "defect"
+
+**training** To train the model for mode `defect`:
+
+    python3 -m scripts.trainer defect
+
```
Comment (marco-c) [inline README.md:79]:
```suggestion

```

**Pull request merged**